ee/institution/admin/saml.sql

```diff
@@ -51,14 +88,28 @@ WITH deleted_authn_provider AS (
     institution_id,
     row_id,
     old_state
-  ) SELECT ...
-), audit_logs_deleted_saml_provider (
+  ) SELECT
+    $authn_user_id as authn_user_id,
+    'institution_authn_providers' as table_name,
+    'delete' as action,
+    $institution_id as institution_id,
+    dap.id as row_id,
+    to_jsonb(dap) AS old_state
```

Comment:
Note that the matching between insert columns and values is by order, not name, so the `AS ...` entries above are not necessary. I'm happy to have them as documentation if that was your intention.

Reply:
Ah fair - I'll remove them!

Reply:
Removed in eaa756d.